src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,25 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    organizationFields.title = organizationFields.name.replace(/\W/g, ''); // eslint-disable-line no-param-reassign
+    apiClient.type('organizations').get(this.props.organization.id)
```

Comment:
Getting the organization again shouldn't be necessary. The org is already fetched in the constructor, so you should be able to do `this.props.organization.update(fieldsToUpdate).save()`